Sources/Core/API/CameraEventCollection.swift

```diff
@@ -10,4 +10,6 @@ import Foundation
 public final class CameraEventCollection: EventCollection<Camera> {
     /// Event triggered when the camera was moved
     public private(set) lazy var moved = Event<Camera, (old: Point, new: Point)>(object: self.object)
+    /// Event triggered when the camera was resized
+    public private(set) lazy var resized = Event<Camera, Void>(object: self.object)
```

Comment:
Looks good! Maybe to be consistent with the moved event the resized event should contain the old and new size? What do you think @JohnSundell

Reply:
Yeah that's a great idea 👍 But I think we can do that in a separate PR that both updates this event and the same event for `Actor` (because we also want consistency there).